Sources/VitaminUIKit/Components/Snackbar/README.md

```diff
@@ -0,0 +1,174 @@
+anotherSnackbar.title = "Another short title is best",
+anotherSnackbar.message = "A message should be shorter",
+anotherSnackbar.maxWidth = 200.0
+```
+
+
+### Displaying a Snackbar
+If you want to manaully display the snackbar, the `VitaminSnackbar` provides a `display()` function, that will first set opacity of the snackbar (and its subviews) to 0, then fade it in by animating the opacity to 1.
+
+```swift
+// let's set properties in constructor
+let snackbar = VitaminSnackbar(
+    basicConfiguration: VitaminSnackbar.BasicConfiguration(
+        title: "A short title is best",
+        message: "A message should be short",
+        image: VitaminAssets.Placeholder.landscape.image
+    )
+)
+
+// Define here the layout constraints to place your snackbar on the screen
+// for example:
+snackbar.centerXAnchor.constraint(equalTo: self.view.centerXAnchor).isActive = true
+snackbar.bottomAnchor.constraint(
+    equalTo: self.view.bottomAnchor,
+    constant: 10.0
+).isActive = true
+
+// fade you snackbar in
+snackbar.display()
+```
+
+⚠️ This method should only be used before displaying the snackbar, because if it is already displayed, it will, disappear, then reappear with a fade in effect.
+
+Since only one snackbar is alowed at the same time, if you display manually the snackbar, it's up to you to remove any other snackbar before (see below)
+
+
+### Dismissal of a Snackbar
+A `VitaminSnackBar` has also advanced properties to handle the dismissal
+- a `autodismiss` boolean: if set to `true`, the `VitaminSnackbar` will be automatically dismissed after a delay. If set to `false`, it will remain displayed until any manual dismissal
+- a `dismissDelay` Double, that represents the number of seconds before the automatic dismissal. Default value is 5. This property has no effect if `autoDismiss` is set to `false`
+- a `dismissOnTap` boolean: if set to `true`, a tap on the snackbar will dismiss it. If set to `false`, snackbar will remain displayed until automatic or programmatic dismissal
+
+All these properties can be set in constructor, by passing them in a `VitaminSnackbar.DismissConfiguration` struct.
+They can also be set separately afterwards
+
+```swift
+// let's set properties in constructor
+// once displayed, snackbar will automatically be dismissed after 10 seconds, or if the user taps on it before
+let snackbar = VitaminSnackbar(
+    basicConfiguration: VitaminSnackbar.BasicConfiguration(
+        title: "A short title is best",
+        message: "A message should be short",
+        image: VitaminAssets.Placeholder.landscape.image
+    ),
+    dismissConfiguration: VitaminSnackbar.DismissConfiguration(
+        autoDismiss: true,
+        dismissDelay: 10.0,
+        dismissOnTap: true
+    )
+)
+
+// or let's define an empty VitaminSnackbar and set its properties afterwards
+// once displayed, snackbar2 will automatically be dismissed after 5 seconds (default value), but tap on it will have no effect
```

Comment:
```suggestion
// once displayed, anotherSnackbar will automatically be dismissed after 5 seconds (default value), but tap on it will have no effect
```